### ISSUE TITLE
Allow editing of assessed applications

### ISSUE THREAD

**Issue by trevorquinn** [redhat-cop/pathfinder-ui#6]
It would be good to be able to open the questionnaire for an existing application, view existing answers, make changes to those answers to revise the assessment, and re-save.

**Comment by texasbrok** [comment]
@noelo or @boogiespook, Kerrie thinks this might be complete. Please confirm and, if so, close.

**Comment by noelo** [comment]
No it's not complete.

**Comment by texasbrok** [comment]
Thanks, @noelo.  I'm also seeing https://github.com/redhat-cop/pathfinder/issues/25.  Are these duplicates?  If so, would you kindly close one of them, please?

**Comment by matallen** [comment]
Yes it is a duplicate of #25. Closed #25 in favor of this issue.

Noel's text on #25 was: Enable the user to modify the survey answers by pre-populating the survey with the previous answers
